cmd/controller/velerovolume/controller.go

```diff
@@ -284,6 +307,24 @@ func (c *Controller) addBackupAnnotationsToPod(pod *corev1.Pod) error {
 	return nil
 }
 
+// removeBackupAnnotationsFromPod removes relevant backup annotation from pod.
+func (c *Controller) removeBackupAnnotationsFromPod(pod *corev1.Pod) error {
+	if pod.Annotations != nil {
+		if _, ok := pod.Annotations[constants.VELERO_BACKUP_ANNOTATION_KEY]; ok {
+			// Remove annotation as it is present
+			podCopy := pod.DeepCopy()
+			delete(podCopy.Annotations, constants.VELERO_BACKUP_ANNOTATION_KEY)
+
+			// Update pod
+			_, err := c.kubeclientset.CoreV1().Pods(pod.Namespace).Update(context.TODO(), podCopy, metav1.UpdateOptions{})
+			if err != nil {
+				return err
+			}
+			klog.V(4).Infof("remove velero restic backup annotation: '%s' to pod '%s/%s' successfully", constants.VELERO_BACKUP_ANNOTATION_KEY, pod.Namespace, pod.Name)
```

Comment:
klog.V(4).Infof("remove velero restic backup annotation: '%s' from pod '%s/%s' successfully", constants.VELERO_BACKUP_ANNOTATION_KEY, pod.Namespace, pod.Name)

Reply:
Done.
I also reported for log when an error occurs during removal.